book/src/filters.md

```diff
@@ -18,9 +18,29 @@ Enable it with Cargo features (see below for more information).
 ## Built-In Filters
 [#built-in-filters]: #built-in-filters
 
+Built-in filters that take (optional) arguments can be called with named arguments, too.
+The order of the named arguments does not matter, but named arguments must come after
+positional (i.e. unnamed) arguments.
+
+E.g. the filter `pluralize` takes two optional arguments: `sg` and `pl` (singular and plural),
```

Comment:
Btw, why not using the full name instead of `sg`/`pl`?

Reply:
The words are just too long, and `sg` and `pl` are universally understood abbreviations, I think.

Reply:
It's the first time I ever saw them, so I'm definitely not convinced. ^^'

Reply:
How about <https://en.wikipedia.org/wiki/List_of_glossing_abbreviations>, <https://en.wiktionary.org/wiki/singular>, <https://en.wiktionary.org/wiki/plural>, <https://en.wikipedia.org/wiki/Goose>?

Reply:
Thanks!